### DISCUSSION TITLE
Test on December 15, 2022

### DISCUSSION THREAD

**Discussion by WimDeMeester** [WimDeMeester/eFinder#41] [Show and tell (success stories and failures)]
Hi all,

Yesterday, we had a new clear night in Belgium and I could test the eFinder again.  
I tested the eFinder again, this time, I both used the latest version from the main branch and the latest release (16.3), which was tested by @AstroKeith.  I only tried the VNC GUI version.  I selected objects both from the Nexus DSC Pro directly and from SkySafari.  The results were the same.  

I will start with the good news:
- The problem with the alignment is fixed.  I use the option in my Nexus DSC Pro to do two star alignment without alt ref.  This makes it very fast to do the initial alignment.  

The not so good news, is that I still experience the problems I had during my first tests on November 13:
- As far as I can tell, the normal Nexus goto worked quite well.  I guess this is because the initial alignment is done way more precisely than when the alignment is done manually.  So, when I go to a new object, this object is most of the time in the FoV of my 21mm Ethos (53').  Using the 'Align via GoTo' option in the GUI, I have the impression that the object is moved a bit more to the centre, but only slightly.  When the object was not in the field of view, using the 'Align via GoTo' did put the object in the FoV.
- When doing a GoTo to Polaris, Polaris was also not centered in the FoV.  I did measure and use the offset.  When I recentered Polaris and measured the offset again, the numbers were very similar to the ones I measured before (I have a quite large offset of around 40, 40).  

Some things I'm wondering about (Maybe you have some answers @AstroKeith):
- Is the offset not applied correctly?
- Is there a problem with how I connect the eFinder to the telescope?  Maybe the offset changes when I move the telescope around (I will check this tonight - inside, because it will be cloudy)...

Any ideas?

**Comment by AstroKeith** [comment]
Getting better then!

I’ve looked earlier at Implementation of the offset and cant find fault. Plus it works for me! 

I suggest next time doing a few additional things,

Using the GUI, make sure the graticule is displayed and check that a) it shows Polaris on initial measurement ( you will need to ‘redraw’) and b) check that later on it points to where the scope is looking (by simple comparison).

After a goto++, what does the GUI say the delta is with respect to target, and with respect to Nexus? On a typical session for me, after the Nexus or SkySafari goto it will report 10 or so arc minutes, after goto++ it will report ~1 arc minute. Although I normally use the handpad!

I’ve seen some ServoCats that cant get closer than 10 arc minutes, due to bad backlash performance. This can be due to sloppy mechanics, or wrong backlash parameters being stored. The above should indicate if this is an issue.

The principle of the offset correction is quite simple. It uses the wcs-xy2rd function. During Polaris measurement, the brightest star is found in the image, and its x & y position stored. The name of the star is displayed so you can check it found the star you wanted. The graticule will also move to the star (on redraw).

So that x,y defines the main scope sightline.

During any subsequent solve, the apply offset process looks at the capture.wcs file created by the solve and simply asks for the RA & Dec of the x & y offset.

So if you have moved any of these functions, check the variables are global if not passed as arguments.

Main thing is to look at the displayed deltas.

Keith

**Reply by WimDeMeester** [reply]
Thanks @AstroKeith , I'll do the needed tests next time.  First, I'll check if the needed global variables are set, but I don't think that will be the problem as I have the same problem with the older code... I'll check all the output on the screen, and probably make same Screencasts too.

**Comment by AstroKeith** [comment]
Something to check….

Make sure that capture.axy and capture.wcs in /Solver/images are being re-written each solve, especially for Polaris calibration.

Keith

**Reply by WimDeMeester** [reply]
Thank, I'll do so!

**Reply by WimDeMeester** [reply]
Hi @mrosseel, 
capture.axy and capture.wcs are written in /dev/shm/images.  I guess that should not be a problem?
